module_evaluations/TCR-32_2023-09-06.MD

```diff
@@ -0,0 +1,79 @@
+# Module acceptance criteria template
+
+## How to use this form
+When performing a technical evaluation of a module, create a copy of this document and use the conventions below to indicate the status of each criterion.  The evaluation results should be placed in the [module_evaluations](https://github.com/folio-org/tech-council/tree/master/module_evaluations) directory and should conform to the following naming convention: `{JIRA Key}_YYYY-MM-DD.MD`, e.g. `TCR-1_2021-11-17.MD`.  The date here is used to differentiate between initial and potential re-evaluation(s).  It should be the date when the evaluation results file was created.
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/7b10294a5c1c10c7e1a7c5b9f99f04bf07630f06/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+## Shared/Common
+* [x] Uses Apache 2.0 license
+* [x] Module build MUST produce a valid module descriptor
+* [ ] Module descriptor MUST include interface requirements for all consumed APIs
+  * missing `edge-fqm`
+  * missing `mod-fqm-manager` (NB: this interface appears to be misnamed; it should not have the `mod-` prefix)
+* [x] Third party dependencies use an Apache 2.0 compatible license
+  * several dependencies licenses are undefined, though not strictly speaking incompatible
+* [x] ~Installation documentation is included~
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+  * It isn't clear to me that personal data is actually stored/cached in cookies, but they are being transparent about the kinds of data this module handles.
+* [x] Sensitive and environment-specific information is not checked into git repository
+* [x] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
+  * In fact there is no list of officially supported front-end languages. This module is written in TypeScript, which is uncommon among front-end modules, but which is nonetheless specifically supported by the UI build tools.
+* [ ] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+  * TCRs are open for all three modules leveraged by this one.
+* [ ] Module gracefully handles the absence of third party systems or related configuration
+  * I was unable to successfully run the module locally and therefore cannot evaluate this
```

Comment:
Considering this "gracefully" question.  Mostly yes.  Creating a dummy list, the UI is a bit odd in saying "No loaded" -- I'm not sure what that means, maybe just missing a word.
![image](https://github.com/folio-org/tech-council/assets/31278545/84c9e586-ff48-4fa5-8228-f6582eb80d75)

FWIW the ui-lists version today on snapshot seems to fix this, but I'm evaluating the specified commit d69e25c.  Maybe just my build.